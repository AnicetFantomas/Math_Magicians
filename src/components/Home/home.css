* {
  padding: 0;
  margin: 0;
  box-sizing: border-box;
}

h2 {
  padding-bottom: 30px;
}

.home-container {
  background-image: url('https://www.skybounddigital.com/wp-content/uploads/2015/01/skills-background1.jpg');
  background-size: cover;
  width: 100%;
  color: #fff;
  font-weight: bold;
  padding: 10% 5% 25% 5%;
  font-size: 20px;
}
