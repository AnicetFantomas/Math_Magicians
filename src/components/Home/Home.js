import React from 'react';
import './home.css';
import Navigation from '../Navigations/Navigation';

const Home = () => (
  <div>
    <Navigation />
    <div className="home-container">
      <h2>Welcome to our page !</h2>
      <p>
        The math magician web app is a simple
        react app with a simple calculator and a
        navigation bar that help to start with React.
        <br />
        Lorem, ipsum dolor sit amet consectetur adipisicing elit.
        Fuga nemo veritatis optio. Fuga voluptatibus,
        dolorem error quo aspernatur repudiandae natus unde deleniti voluptate,
        mollitia quibusdam illo inventore nisi dolor maxime.
        Aperiam natus exercitationem illum! Maiores exercitationem fuga molestiae,
        rerum dignissimos incidunt inventore corrupti laboriosam qui totam reiciendis
        accusamus vel ea aliquid odio nostrum tenetur, corporis perspiciatis cum,
        et quae aspernatur?
        Suscipit placeat repudiandae nostrum, nulla ea natus nesciunt iste, perspiciatis repellat,
        totam eveniet.
      </p>
    </div>
  </div>
);

export default Home;
