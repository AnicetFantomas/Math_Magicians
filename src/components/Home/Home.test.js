import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { BrowserRouter as Router } from 'react-router-dom';
import Home from './Home';

describe('Home page of App', () => {
  test('Build the snapshot of the Home component safely', () => {
    const home = render(
      <Router>
        <Home />
      </Router>,
    );
    expect(home).toMatchSnapshot();
  });
});
