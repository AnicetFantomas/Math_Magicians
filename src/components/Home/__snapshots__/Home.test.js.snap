// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page of App Build the snapshot of the Home component safely 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="Navigation-bar"
        >
          <h1>
            Math Magician
          </h1>
          <div
            class="navLinks"
          >
            <a
              activeclassname="nav-active"
              aria-current="page"
              class="link-el active"
              href="/"
            >
              Home
            </a>
            <a
              activeclassname="nav-active"
              class="link-el"
              href="/Calculator"
            >
              Calculator
            </a>
            <a
              activeclassname="nav-active"
              class="link-el"
              href="/Quotes"
            >
              Quotes
            </a>
          </div>
        </div>
        <div
          class="home-container"
        >
          <h2>
            Welcome to our page !
          </h2>
          <p>
            The math magician web app is a simple react app with a simple calculator and a navigation bar that help to start with React.
            <br />
            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Fuga nemo veritatis optio. Fuga voluptatibus, dolorem error quo aspernatur repudiandae natus unde deleniti voluptate, mollitia quibusdam illo inventore nisi dolor maxime. Aperiam natus exercitationem illum! Maiores exercitationem fuga molestiae, rerum dignissimos incidunt inventore corrupti laboriosam qui totam reiciendis accusamus vel ea aliquid odio nostrum tenetur, corporis perspiciatis cum, et quae aspernatur? Suscipit placeat repudiandae nostrum, nulla ea natus nesciunt iste, perspiciatis repellat, totam eveniet.
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="Navigation-bar"
      >
        <h1>
          Math Magician
        </h1>
        <div
          class="navLinks"
        >
          <a
            activeclassname="nav-active"
            aria-current="page"
            class="link-el active"
            href="/"
          >
            Home
          </a>
          <a
            activeclassname="nav-active"
            class="link-el"
            href="/Calculator"
          >
            Calculator
          </a>
          <a
            activeclassname="nav-active"
            class="link-el"
            href="/Quotes"
          >
            Quotes
          </a>
        </div>
      </div>
      <div
        class="home-container"
      >
        <h2>
          Welcome to our page !
        </h2>
        <p>
          The math magician web app is a simple react app with a simple calculator and a navigation bar that help to start with React.
          <br />
          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Fuga nemo veritatis optio. Fuga voluptatibus, dolorem error quo aspernatur repudiandae natus unde deleniti voluptate, mollitia quibusdam illo inventore nisi dolor maxime. Aperiam natus exercitationem illum! Maiores exercitationem fuga molestiae, rerum dignissimos incidunt inventore corrupti laboriosam qui totam reiciendis accusamus vel ea aliquid odio nostrum tenetur, corporis perspiciatis cum, et quae aspernatur? Suscipit placeat repudiandae nostrum, nulla ea natus nesciunt iste, perspiciatis repellat, totam eveniet.
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
